data/remnant/remnant missions.txt

```diff
@@ -3466,6 +3466,45 @@ event "remnant: cognizance timer"
 
 
 
+mission "Remnant: Cognizance 19"
+	name "Deliver nav data"
+	description "Deliver navigational data on the new systems to <destination>."
+	landing
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Cognizance 18: done"
+	on offer
+		conversation
+			`As you finish your landing checks, your comlink notifies you of an incoming message from Dawn:`
+			`	"Greetings, Captain <first>! We will be compiling updated navigational charts for the new systems and would like a copy of the data your nav system recorded. Stop by <planet> and the starport systems will copy the data during the systems check. It's a pretty small data set, so there's no need to use data crystals. Thank you!"`
+			`	The end of the message is the location of a starport on <planet>, presumably where you are supposed to go.`
+				accept
+	on complete
+		payment 100000
+		dialog `As the starport computer downloads the data, your computer displays a message tacked onto the log file: "Thank you for providing your navigational logs. They will be useful to contrast against those of our scouts. A logistics adjustment of <payment> has been authorized for the <ship>."`
```

Comment:
```suggestion
		dialog `As the starport computer downloads the data, your computer displays a message tacked onto the log file: "Thank you for providing your navigational logs. They will be useful to compare with those of our scouts. A logistics adjustment of <payment> has been authorized for the <ship>."`
```
`contrast against` is fine, I just personally prefer `compare with`. Obviously it sounds like those are antonyms, but they essentially mean the same thing: both imply you're looking for similarities and differences in two "objects". I guess `contrast against` prioritizes differences, while `compare with` prioritizes similarities. If you think `contrast against` is more what you're going for, then feel free to resolve. Up to you.

Reply:
She'd be looking for both, but differences are probably more intriguing to her: What has changed between your first-on-the-scene info vs the subsequent scouts?